### DISCUSSION TITLE
hub_server can not lanuch web page

### DISCUSSION THREAD

**Discussion by StevenLeiZhang** [artifacthub/hub#1768] [Q&A] [answered]
I follow dev doc  to set up my local dev environment , but the web page does now render normally .

procedurce :
```
cd hub
hub_server
```

hub_server does not print any error, but when I open http://localhost:8000/ with chrome, there web page is not rendered as bellow:

![80374A4E-98BE-4479-81BB-D85488E385D7](https://user-images.githubusercontent.com/14890978/147314295-79e2e940-15c9-4155-bb62-e838c6d7f1b3.jpg)

but when I run "hub_frontend_dev", the web page works well.
![7E6FB68E-6DA5-49C3-A085-4FCF4C34F183](https://user-images.githubusercontent.com/14890978/147314354-db83f670-aab1-4a53-bac1-f122ab9382a3.jpg)

I am not frontend develper, so I don't know how to fix it . Anyone know the reason why hub_server can not lanuch web page?

**Comment by tegioz** [comment, marked as answer]
Hi @StevenLeiZhang 👋 

The hub configuration displayed in the dev environment setup docs [was missing some entries](https://github.com/artifacthub/hub/pull/1769/files) that may be causing this issue. Could you please give it a try again after adding them?

**Reply by StevenLeiZhang** [reply]
Thank you Sergio. After adding them, my local web page works well now!

**Reply by tegioz** [reply]
Awesome, no worries 👍
